plugins/gatsby-remark-header-custom-ids/index.js

```diff
@@ -35,33 +32,43 @@ module.exports = (
       lastNode.value = lastNode.value.replace(match[1], '');
     }
 
-    const data = patch(node, 'data', {});
```

Comment:
I'm not sure about changing this. Can we get back the old logic?

Reply:
This code not necessary and not work :)

Reply:
Let's just leave these things as they used to be, as well as that new line :up:. I just want to have changes that solve the current problem.

`*.md` files and `yarn.lock` are marked as modified because you created your PR from the `master` branch. Let's fix this too.

Reply:
Ok, I will prepare a new Pull Request